posts/mpox-preparedness/index.qmd

```diff
@@ -0,0 +1,100 @@
+:::
+
+#### Population level ($R0$ and $Rt$)
+
+The basic reproduction number, $R0$, is the average number of secondary cases produced by a single infected individual in a completely susceptible population. The instantaneous reproduction number, $Rt$, on the other hand, is the average number of secondary cases produced by a single infected individual at time $t$ in a partially susceptible population. The instantaneous reproduction number is a more useful quantity during an outbreak as it accounts for the impact of interventions and changes in population immunity.
+
+If data is available on the daily number of reported cases, [`{EpiNow2}`](https://epiforecasts.io/EpiNow2/) and  [`{EpiEstim}`](https://mrc-ide.github.io/EpiEstim/) can be used to estimate the time varying reproduction number, $Rt$. These packages require data on the time scale of transmission (i.e. the generation time, or the serial interval, which is [commonly used as a proxy](https://royalsocietypublishing.org/doi/10.1098/rsif.2020.0756) for this). While [`{EpiEstim}`] focuses on retrospective estimation of $Rt$, `{EpiNow2}` is designed for both retrospective and real-time estimation.
+
+In estimating $Rt$, one practical consideration is the impact of various delays (biological and reporting) on the estimates [@charniga2024best;@park2024estimating;@gostic2020practical]. `{EpiNow2}` adjusts for these delays in various ways. For example, it accounts for the symptom onset and reporting delays by taking the incubation period and reporting delay as inputs. Moreover, `{EpiNow2}` can estimate estimate the reporting delay from the data if data on incidence by date of onset and report are available.
+
+Furthermore, dedicated packages have emerged for estimating epidemiological delays from data using best practices. [`{epidist}`](https://epidist.epinowcast.org/articles/epidist.html) offers the ability to estimate delay distributions, accounting for issues such as truncation (i.e., not all disease outcomes will yet be known in real-time).
+
+If delay data are not available, published estimates of the incubation period and serial interval can be used. The [`{epiparameter}`](https://epiverse-trace.github.io/epiparameter/) package collates a database of epidemiological distributions from the literature and provides functions for interacting with the database. See the [database here](https://epiverse-trace.github.io/epiparameter/dev/articles/database) for currently available parameters (more entries are planned). Additionally, if only summary statistics are available (e.g. range and median), `{epiparameter}` can be used to extract the distribution parameters.
```

Comment:
Ths epiparameter database link will likely need to be updated after https://github.com/epiverse-trace/epiparameter/issues/336.

Reply:
It still works as it doesn't use the .dev domain.